libraries/hwlib/library/targets/hwlib-native.hpp

```diff
@@ -13,24 +13,50 @@
 // this file contains Doxygen lines
 /// @file
 
+#ifdef __unix__                    /* __unix__ is usually defined by compilers targeting Unix systems */
+
+    #define OS_Windows 0
```

Comment:
Is it not better to use 

```cpp
#define OS_Linux
    #include <sys/time.h>
    #define EPOCH_DIFF 11644473600LL
 #elif defined(_WIN32) || defined(WIN32) /* _Win32 is usually defined by compilers targeting 32 or   64 bit Windows systems */
    #define OS_Windows
    #include <windows.h>
 #endif

```

Reply:
The indents on the lines after this one seem a bit out of line as well

Reply:
yes the define could be removed, and will look at the indents